pkg/cidata/cidata.TEMPLATE.d/boot/00-check-rtc-and-wait-ntp.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+set -eu
+
+# In vz, the VM lacks an RTC when booting with a kernel image (see: https://developer.apple.com/forums/thread/760344).
+# This causes incorrect system time until NTP synchronizes it, leading to TLS errors.
+# To avoid TLS errors, this script waits for NTP synchronization if RTC is unavailable.
+test ! -c /dev/rtc0 || exit 0
+
+# This script is intended for services running with systemd.
+command -v systemctl >/dev/null 2>&1 || exit 0
+
+echo_with_time_usec() {
+	time_usec=$(timedatectl show --property=TimeUSec)
+	echo "${time_usec}, ${1}"
+}
+
+# For the first boot, where the above setting is not yet active, wait for NTP synchronization here.
+max_retry=60 retry=0
+until ntp_synchronized=$(timedatectl show --property=NTPSynchronized --value) && [ "${ntp_synchronized}" = "yes" ] ||
+	[ "${retry}" -gt "${max_retry}" ]; do
+	if [ "${retry}" -eq 0 ]; then
+		# If /dev/rtc is not available, the system time set during the Linux kernel build is used.
+		# The larger the difference between this system time and the NTP server time, the longer the NTP synchronization will take.
+		# By setting the system time to the modification time of this script, which is likely to be closer to the actual time,
```

Comment:
cidata/user-data might be closer

Reply:
When running this script, sub-second precision of the file's modification time is irrelevant since NTP synchronization is assumed. Using the script file itself ensures the file reliably exists.

Reply:
Potentially, a future version of Lima may install the cidata script without go:embed.

In that case, the timestamp of the cidata script will corresponds to the timestamp in the git repo, which may be years behind the current clock

Reply:
changed to use `user-data`